common/src/main/java/com/distkv/dst/common/exception/SortedListIncrScoreOutOfRange.java

```diff
@@ -0,0 +1,29 @@
+package com.distkv.dst.common.exception;
+
+public class SortedListIncrScoreOutOfRange extends DstException {
```

Comment:
What's the purpose of this class?

Reply:
When we are increasing the score of the member by **incrScore** method,  the score which is integer out of bounds may occur. For example, the original score is 10, when we plus 2147483647  (that is **Integer.MAX_VALUE**). It is outing of range so that we must catch the exception.

Reply:
Nice